### PR TITLE
Decreases support-annotation version to 22.0.0 to be able to compile …

### PR DESCRIPTION
**Pull request by vogella**
To compile the "BasisSample" with Gradle I had to adjust the version of the support-annotations to 22.0.0.

### CODE REVIEW

**Comment by vogella** [comment]
I signed it!

**Comment by vogella** [comment]
I signed it!

On Fri, May 15, 2015 at 5:27 PM, googlebot <notifications@github.com> wrote:

> Thanks for your pull request. It looks like this may be your first
> contribution to a Google open source project, in which case you'll need to
> sign a Contributor License Agreement (CLA).
>
> [image: :memo:] *Please visit https://cla.developers.google.com/
> <https://cla.developers.google.com/> to sign.*
>
> Once you've signed, please reply here (e.g. I signed it!) and we'll
> verify. Thanks.
> ------------------------------
>
>    - If you've already signed a CLA, it's possible we don't have your
>    GitHub username or you're using a different email address. Check your
>    existing CLA data <https://cla.developers.google.com/clas> and verify
>    that your email is set on your git commits
>    <https://help.github.com/articles/setting-your-email-in-git/>.
>    - If you signed the CLA as a corporation, please let us know the
>    company's name.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/googlesamples/android-testing/pull/24#issuecomment-102432315>
> .
>



-- 
Geschäftsführer

vogella GmbH

Haindaalwisch 17a, 22395 Hamburg
Amtsgericht Hamburg: HRB 127058
Geschäftsführer: Lars Vogel, Jennifer Nerlich de Vogel
USt-IdNr.: DE284122352
Fax (032) 221739404, Email: lars.vogel@vogella.com, Web:
http://www.vogella.com

File: espresso/BasicSample/app/build.gradle [hunk 1]
```diff
@@ -24,7 +24,7 @@ android {
 
 dependencies {
     // App dependencies
-    compile 'com.android.support:support-annotations:22.1.0'
```
Comment (otw) [inline espresso/BasicSample/app/build.gradle]: Revert and add:
allprojects {
    repositories {
        jcenter()
    }

    configurations.all {
        resolutionStrategy.force 'com.android.support:support-annotations:22.1.0'
    }
}

to espresso/BasicSample/build.gradle

Would be great if you could apply the same fix to all the individual samples which depend on support-annotations

**Comment by otw** [comment]
I added a comment to your change on how to fix it. Basically the issue is the we force resolution for the top lvl build file but not for each sample. We didn't catch this regression on the build server because we always build from the root folder.

**Comment by vogella** [comment]
@otw Started to fix these issues with your advise. Please see https://github.com/googlesamples/android-testing/pull/29

**Pull request closed without merging**